app/src/main/java/vn/mbm/phimp/me/newGallery.java

```diff
@@ -3445,7 +3448,35 @@ public void onResume()
               }catch(Exception e){
 
               }
-            
+            localPhotosGrid.setOnScrollListener(new AbsListView.OnScrollListener() {
+                @Override
+                public void onScrollStateChanged(AbsListView view, int scrollState) {
+
```

Comment:
OnScroll is currently appearing as an unwanted function now as it doesnot do anything. I asked what the empty line was! Sorry of you felt it rude, it was not meant to be so :)